test/StandardERC721.js

```diff
@@ -0,0 +1,17 @@
+// const expectThrow = require('./utils').expectThrow
+const TestERC721ImplementationAbstraction = artifacts.require('TestERC721Implementation')
+let ERC721
+
+contract('TestERC721Implementation', function (accounts) {
+  beforeEach(async () => {
+    // todo: deployment is OOG-ing.
```

Comment:
Not OOGing is failing at compile time for not implementing all the function definitions on the interface.